superduper/backends/base/query.py

```diff
@@ -171,6 +226,8 @@ class Query(_BaseQuery):
     identifier: str = ''
 
     def __getitem__(self, item):
+        if isinstance(item, str):
```

Comment:
how is 
query = db['documents'].find({})

query['some_item'] a condition?

Reply:
Yes

select = table.select("x", "y", "n").filter(table["y"] == 1, table['n'] > 5)

Reply:
ibis syntax is 
table.y 
does ibis support the indexing syntax as well>
table["y"] ?

Reply:
Yes, In the latest update, a new usage method was added for ibis without changing the original logic.